tm2py/components/component.py

```diff
@@ -62,31 +90,23 @@ def controller(self):
         """Parent controller."""
         return self._controller
 
+    @property
+    def emme_manager(self):
+        return self.controller.emme_manager
+
```

Comment:
Adding this here so we don't have to have it everywhere this method is implemented.